src/index.js

```diff
@@ -1,18 +1,31 @@
 import 'assets/styles/main.scss';
-import cloudinary from 'cloudinary-core';
 import VideoPlayer from './video-player';
 import { assign } from 'utils/assign';
 
-cloudinary.VideoPlayer = VideoPlayer;
 
-const proto = cloudinary.Cloudinary.prototype;
+function videoPlayer(id, playerOptions, ready = null, cloudinaryConfig) {
```

Comment:
What is the use case for the 3rd & 4th arguments?
Do we have an example using them?

Reply:
the 4th argument was for internal use. I was rewrite it